.github/workflows/release.yml

```
@@ -0,0 +1,19 @@
+name: CI (develop)
```

nameがdevelop.ymlと重複しているため変更してほしい
```suggestion
name: CI (release)
```